tests/components/fritz/test_config_flow.py

```diff
@@ -282,7 +282,10 @@ async def test_exception_connection(
     [(True, MOCK_USER_INPUT_ADVANCED), (False, MOCK_USER_INPUT_SIMPLE)],
 )
 async def test_exception_unknown(
-    hass: HomeAssistant, mock_get_source_ip, show_advanced_options: bool, user_input
+    hass: HomeAssistant,
+    mock_get_source_ip: None,
```

Comment:
Better example

Reply:
I have reverted the changes with `None` type hints.
I need to think some more how to resolve this with the pylint plugin, and will work on those in a follow-up PR.